README.md

```diff
@@ -1,80 +1,68 @@
-# ubuntu-advantage tool
+# Ubuntu Advantage Client
 
-[![Build Status](https://travis-ci.org/CanonicalLtd/ubuntu-advantage-script.svg?branch=master)](https://travis-ci.org/CanonicalLtd/ubuntu-advantage-script)
+[![Build Status](https://travis-ci.org/CanonicalLtd/ubuntu-advantage-client.svg?branch=master)](https://travis-ci.org/CanonicalLtd/ubuntu-advantage-client)
 
-This tool is used to enable or disable specific Ubuntu Advantage offerings from Canonical. Version 18.1 or later is a python-based client which talks to a
-UA Contract backend service API to navigate UA contracts, entitlement details
-and status.
+The Ubuntu Advantage client provides users with a simple mechanism to
+view, enable, and disable offerings from Canonical on their system. The
+following entitlements are supported:
 
-Currently it supports setup and maintenance of the following entitlements:
+- [Ubuntu Extended Security Maintenance](https://ubuntu.com/esm)
+- [Livepatch Service](https://www.ubuntu.com/livepatch)
+- FIPS 140-2 Certified Modules
+- FIPS 140-2 Non-Certified Module Updates
+- Common Criteria EAL2 certification artifacts provisioning
```

Comment:
Since we are listing this in the readme, I've opened a corresponding issue #175  to add this support to the new uaclient.